packages/mux-player/src/index.ts

```diff
@@ -786,9 +786,6 @@ class MuxPlayerElement extends VideoApiElement {
    * Set the beacon collection domain. Used by Mux Data.
    */
   set beaconCollectionDomain(val: string | undefined) {
-    // don't cause an infinite loop
-    if (val === this.beaconCollectionDomain) return;
```

Comment:
mux-player doesn't have this issue, the infinite loop is related to CustomVideoElement.js